src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ValueDistFunction.java

```diff
@@ -95,8 +93,7 @@ public double setUtility(double currentdistance) {
 	@Override
 	public double getSubjectiveValue(LatLng objectiveData) {
 		if (interestlocation.containsKey(objectiveData) == false) {
```

Comment:
Il est moins couteux (à petite echelle) de le remplacer par :
`!interestlocation.containsKey(objectiveData)`
;) 

_Note (sous le contrôle de @oliviercailloux ) : 
Cela est normalement plus efficace car dans le cas présent : 
Comparaison de la valeur récupéré avec la valeur false :

- vérification des instances 

- comparaison des valeurs

Dans l'autre cas : 

-  Inversion de la valeur (moins couteux)

Reply:
Fixed in commit ea9579e